docs/api/cloudmarker.stores.rst

```diff
@@ -9,14 +9,6 @@ cloudmarker.stores package
 Submodules
 ----------
 
-cloudmarker.stores.emailalert module
-------------------------------------
-
-.. automodule:: cloudmarker.stores.emailalert
-    :members:
-    :undoc-members:
-    :show-inheritance:
-
 cloudmarker.stores.filestore module
 -----------------------------------
```

Comment:
The `emailalert` documentation should appear in `cloudmarker.alerts.rst`, so if this deletion change is included in this PR, the corresponding addition change should also be included in this PR.

If you just run `make docs`, that should do the necessary updates to make these changes ready.

Reply:
Done